### PR TITLE
Output progress and results as attachments

### PR DESCRIPTION
**Pull request by jkeiser**
- Don't display warnings and informational messages in channel
- Display a "progress message" while handlers are running that streams the logs (only streams in PM)
- Display success and error messages as attachments so they have color and are distinguishable from other messages
- Link to full logs on error and in progress

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,5 +7,5 @@ group(:style) do
 end
 
 group(:slack) do
-  gem "lita-slack"
+  gem "lita-slack", github: "jkeiser/lita-slack", branch: "send-messages-with-arguments"
```
Comment (PrajaktaPurohit) [inline Gemfile]: should this branch get merged first?

Comment (jkeiser) [inline Gemfile]: I don't think so, we can't really block on someone else.

**Comment by PrajaktaPurohit** [comment]
+1 - this is awesome!

**Comment by mwrock** [comment]
:+1:

**Comment by mwrock** [comment]
is this blocked or did you just for get to merge @jkeiser :)

**Pull request merged**